deps.ffmpeg/60-lame.ps1

```diff
@@ -42,14 +43,20 @@ function Build {
     $BuildMachines = @{
         x64 = 'x64'
         x86 = 'I686'
+        arm64 = 'arm64'
     }
 
     $Params = @{
         BasePath = (Get-Location | Convert-Path)
         BuildPath = "."
-        BuildCommand = "nmake -f Makefile.MSVC MACHINE=/machine:$($BuildMachines[$Target]) COMP=MS ASM=NO MSVCVER=Win64"
+        BuildCommand = "nmake -f Makefile.MSVC MACHINE=/machine:$($BuildMachines[$Target]) MMX=NO COMP=MS ASM=NO MSVCVER=Win64"
```

Comment:
Why the additional `MMX=NO` here?

Reply:
It doesn't compile otherwise (as ARM64 has no MMX obviously) and the performance impact should be negligible on modern CPUs anyway.

Reply:
Fair.